OlenaNehodenko/HW5/HW5.py

```diff
@@ -0,0 +1,5 @@
+#First task: Create a list
+List=[1,2,3,4,5,6]
```

Comment:
good job
look at this version of code
list_of_number = [2, 1, 4, 6, 1, 4, 8, 6]

for i in range(len(list_of_number)):
    list_of_number[i] = float(list_of_number[i])
print(list_of_number)